Week3/homework/step4.js

```diff
@@ -1,8 +1,13 @@
 'use strict';
 
-// Add your code here
+const addSix = function (x) {
+  return function createBase() {
+    return x + 6;
+  };
+};
 
```

Comment:
✅ This does what is asked, however, I would be inclined to create a more generic version.  Imagine you were then asked to create an `add8` function.  You would have to copy a lot of code.
```
const createBase = function(x) {
  return function(y) {
    return x + y;
};
};
const addSix = createBase(6);
const addEight = createBase(8);
console.log(addSix(12)); // ==> 18
console.log(addEight(12)); // ==> 20
```
You can see that if `createBase` is able to receive a parameter, it will create an `add` anything function for you.  This makes for much better reuse, which means a lot less typing in the long run. 😉